### ISSUE TITLE
2.5.3 freeze after entering password

### ISSUE THREAD

**Issue by matthias-zivim** [keepassxreboot/keepassxc#4260]
windows 10 64bit 1903 (10.0.18362.592)
install kp xc v2.5.2 -> update to kp xc v2.5.3

With the version kp xc v2.5.2 the problem did not exist. Only after the update.

If kp xc was open for several hours there is the problem. The program freezes after I enter the password to unlock the database.

![Clipboard 2](https://user-images.githubusercontent.com/60196276/73336738-ebeb6900-4272-11ea-96c7-d37b503620aa.jpg)

**Comment by szi-secunet** [comment]
Same problem here! Also German locale and Windows 10.

**Comment by ceving** [comment]
Same problem here.

**Comment by ceving** [comment]
Link to the old version to downgrade:
https://github.com/keepassxreboot/keepassxc/releases/tag/2.5.2

**Comment by droidmonkey** [comment]
Are you trying to switch tabs while the database is being unlocked? If so that is a known issue.

**Comment by ceving** [comment]
There is no chance to switch any tabs. The program freezes right after entering the password and pressing <kbd>Return</kbd>.

**Comment by jasonbassford** [comment]
I have the same problem on my home computer; strangely, I can't replicate it on my work computer. (I haven't isolated what is different between the two.) So, at work I'm using 2.5.3, but at home I'm using 2.5.2.

**Comment by droidmonkey** [comment]
Align the versions and let us know.

**Comment by jasonbassford** [comment]
Your comment doesn't make any sense. As I've already said, when my home computer is using 2.5.3, I experience exactly what I've experienced here. It freezes immediately after supplying the password. That's why I've reverted it to 2.5.2, where everything works.

**Comment by droidmonkey** [comment]
I'm sorry that's not how your original comment read. Can you please explain the differences between your work KeePassXC and your home KeePassXC. Differences as in the settings applied, OS installed on, method of install (portable vs installed). That could lead to a narrowing of the problem set.

**Comment by droidmonkey** [comment]
This issue is a duplicate of the other freeze issues.

**Comment by explorer79** [comment]
@droidmonkey 

I experience the same problem.
I just installed version 2.5.4 (Windows 10, Installer (64-bit)) in a new virtual machine and wanted to connect to my existing keepass file that is located on my onedrive. I unlock my date base only with a password, but not with a hardware key. But it seems to require a hardware key as you can see that there is a waiting animation below the hardware key field.
![image](https://user-images.githubusercontent.com/67898576/86576769-7c904880-bf79-11ea-804e-10868e5e76cc.png)
When this appears it waits forever even after entering the correct password.

While playing around with the settings I found out that **this error occurs only when the browser integration is activated**. When I deactivate the browser integration, I can log in. But as soon as I activate it again, the error message from @matthias-zivim appears when I close the application and when I try to open my keepass database it doesn't open.
But for me the browser integration is an essential feature. I hope with this information you can easily reproduce and fix the error. Thank you

**Comment by ceving** [comment]
I switched to 2.6.0 for a few days and did not notice a freeze till now.
